tests/aws/services/stepfunctions/v2/test_state/test_test_state_scenarios.py

```diff
@@ -0,0 +1,864 @@
+import json
+
+from botocore.config import Config
+from localstack_snapshot.snapshots.transformer import RegexTransformer
+
+from localstack.aws.api.stepfunctions import InspectionLevel
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.utils.strings import short_uid
+from tests.aws.services.stepfunctions.templates.services.services_templates import (
+    ServicesTemplates as ST,
+)
+from tests.aws.services.stepfunctions.templates.test_state.test_state_templates import (
+    TestStateTemplate as TCT,
+)
+
+
+@markers.snapshot.skip_snapshot_verify(
+    paths=[
+        "$..loggingConfiguration",
+        "$..tracingConfiguration",
+        "$..SdkHttpMetadata",
+        "$..SdkResponseMetadata",
+    ]
+)
+class TestStateCaseScenarios:
+    @staticmethod
+    def _send_test_state_request(aws_client_factory, **kwargs):
```

Comment:
For TestState calls, boto will prepend "sync-" to the endpoint string. As we operate on localhost, this function creates a new client with that functionality disabled, and forwards the test_state call. 
As for now this is limited to TestState calls, my initial idea was to make this issue explicit. Creating a step functions client that always disables this feature I though could hide this issue in other/future instances. Maybe this may result in more efficient customer feedback.
I propose adding a fixture for SFN client but include TestState in the name, to improve readability of the test suite but force future occurrences to handle this issue explicitly.